src/yang/ietf-qos-action.yang

```diff
@@ -364,87 +364,87 @@ module ietf-qos-action {
           "A unique name for named counter.";
       }
       description
-        "the count action";
+        "Named count action container";
     }
     description
-      "the count action grouping";
+      "Named count action grouping";
   }
 
   grouping discard {
     container discard {
       leaf discard {
         type empty;
         description
-          "discard action";
+          "Discard action";
       }
       description
-        "discard action";
+        "Discard action container";
     }
     description
-      "discard grouping";
+      "Discard action grouping";
   }
 
   grouping priority {
     container priority {
       leaf level {
         type uint8;
         description
-          "priority level";
+          "Priority level action";
       }
       description
-        "priority attributes";
+        "Priority action container";
     }
     description
-      "priority attributes grouping";
+      "Priority action grouping";
   }
   grouping min-rate {
     container min-rate {
       uses rate-value-unit;
       description
-        "min guaranteed bandwidth";
+        "Min guaranteed bandwidth container";
     }
     description
-      "minimum rate grouping";
+      "Minimum rate grouping";
   }
   grouping dscp-marking {
     container dscp {
       leaf dscp {
         type inet:dscp;
         description
-          "dscp marking";
+          "DSCP marking";
       }
       description
-        "dscp marking container";
+        "DSCP marking container";
     }
     description
-      "dscp marking grouping";
+      "DSCP marking grouping";
   }
   grouping traffic-group-marking {
     container traffic-group {
       leaf traffic-group {
         type string;
         description
-          "traffic group marking";
+          "Traffic group marking";
```

Comment:
I do not think this "Traffic group marking". At the minimum, this is a unique name for the traffic group.